static/app/views/alerts/rules/details/metricChart.tsx

```diff
@@ -57,7 +57,8 @@ import {
 
 import {TimePeriodType} from './constants';
 
-const X_AXIS_BOUNDARY_GAP = 20;
+const LEFT_PADDING = 30;
+const RIGHT_PADDING = 17;
```

Comment:
Would be nice to get a little comment above these (should be `//` not `/**` since you want to comment the paragraph of code, not a single element

```tsx
// The _PADDING constants are used to contain elements within the
// charts (this comment seems wrong but you get the idea)

const LEFT_PADDING = 30;
const RIGHT_PADDING = 17;
...
```